Makefile

```diff
@@ -58,3 +58,8 @@ go_deps: $(find .  -type f | grep '\.go$' | grep -v '\.pb.go$')
 dev_data:
 	cd $(WPTD_GO_PATH)/util; go get -t ./...
 	go run util/populate_dev_data.go
+
+webapp_deploy_staging:
+	gcloud config set project wptdashboard
+	gcloud auth activate-service-account --key-file $(WPTD_PATH)/client-secret.json
+	cd $(WPTD_PATH)/git; ./git-deploy -q
```

Comment:
What is `git-deploy`? I don't see the file under `git/`.

Reply:
It's moved to `util/deploy.sh` since I originally authored this.